src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+  transition: all 300ms linear;
```

It's better not to use `all `property. It's bad for performance, [here](https://www.pno.dev/articles/dont-use-the-all-keyword-in-css-transitions/) you can read more about it.

It is better to directly indicate the properties that will be animated
```suggestion
  transition: transform, color 300ms linear;
```